devices/IFrameTransformToIWear/conf/iframetransform_to_iwear.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE robot PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+<robot name="IFrameTransformToIWear" build=0 portprefix="">
+
+<device type="transformClient" name="TransformClient">
+    <param name="period">0.5</param>
+    <param name="local">/tf</param>
+    <param name="remote">/transformServer</param>
+</device>
+
+<device type="iframetransform_to_iwear" name="IFrameTransformToIWear">
+
+    <param name="wearableName">TransformServer</param>
```

Comment:
That should be fine, since in this example the source is the transform server, and indeed the sensors name will be something like `TransformServer::pose::/iCub/....`. Anyway this file is meant to be just an example on how to use the device